cloud/aws/templates/aws_oidc/app.tf

```diff
@@ -190,6 +194,7 @@ locals {
               aws_secretsmanager_secret.applicant_oidc_client_id_secret.arn,
               aws_secretsmanager_secret.admin_oidc_client_secret_secret.arn,
               aws_secretsmanager_secret.admin_oidc_client_id_secret.arn,
+              aws_secretsmanager_secret.esri_arcgis_api_token_secret.arn,
```

Comment:
**suggestion:** In some places this is called `esri_arcgis_api_token` without the secret - maybe make it consistent one way or another?

Reply:
This is where I was starting to get confused. Trying to follow the naming convention within terraform, but also not exactly sure what needs to match the environment variable in the app.

The environment variable is named `ESRI_ARCGIS_API_TOKEN`. @dkatzz or @rockycodes, it would help greatly if you could tag the changed lines in this PR to indicate which must match the environment variable name (if any).

Reply:
@gwendolyngoetz I just tagged it, looks like it's already matching! The naming conventions in this repo are pretty inconsistent. I think it would be helpful to start a convention where we name everything the same.

Reply:
Thanks @rockycodes I'll go through after morning meetings and make sure things are completely consistent.